### PR TITLE
add  update  label on the menu to allow the user to take advantage of …

### PR DESCRIPTION
**Pull request by Jukoo**
…the latest updates and make a quick upgrade

    the button will be available only if a recent version is published and allows more autonomy for the user

On branch core_contrib
 Changes to be committed:
    new file:   .gitignore
	renamed:    logo-t.png -> Imgs/logo-t.png
	renamed:    main.png -> Imgs/main.png
	renamed:    menu.png -> Imgs/menu.png
	renamed:    options.png -> Imgs/options.png
	modified:   README.md
	modified:   bashtop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+![bashtop_version](https://img.shields.io/badge/BashTop-v0.8.17-brightgreen)
 
 ## Index
 
 * [Changelog](#changelog)
 * [Description](#description)
 * [Features](#features)
 * [Themes](#themes)
-* [Upcoming](#upcoming-(osx-and-bsd-support))
+* [Upcoming](#upcoming-(osx-and-bsd-support)
```
Comment (aristocratos) [inline README.md:14]: This breaks the link

File: bashtop [hunk 2]
```diff
@@ -132,6 +137,13 @@ menu_quit_selected=(
 "║═╬╗║ ║ ║ ║  "
 "╚═╝╚╚═╝ ╩ ╩  ")
 
+menu_update_selected=(
```
Comment (aristocratos) [inline bashtop:140]: Should be double lined to follow the look of selected item

File: bashtop [hunk 3]
```diff
@@ -132,6 +137,13 @@ menu_quit_selected=(
...
+"*┬ ┬┌─┐┌┬┐┌─┐┌┬┐┌─┐"
+" │ │├─┘ ││├─┤ │ ├┤ "
+" └─┘┴  ─┴┘┴ ┴ ┴ └─┘"
+)
+
+declare -r  remote_src="https://raw.githubusercontent.com/aristocratos/bashtop/master/$(basename $0)"
```
Comment (aristocratos) [inline bashtop:146]: This will fail to get remote if local file isn't named "bashtop"

File: bashtop [hunk 4]
```diff
@@ -425,6 +443,40 @@ quit_() { #? Clean exit
 	exit 0
 }
 
+update_ ()  {  #  update label  menu  to   upgrade  to new version   
+     
+    local -a  root_binary_path=("/bin/" "/usr/bin/")  
+    local  src=$(basename $0)
```
Comment (aristocratos) [inline bashtop:449]: Again will fail to find update if running version isn't named "bashtop"

File: bashtop [hunk 5]
```diff
@@ -425,6 +443,40 @@ quit_() { #? Clean exit
...
+        done 
+    else 
+        path=${src} 
+    fi
+    
+    [[ -n  ${path}  ]] && {
+         curl  ${remote_src}  -o ${path} 
+         test $? -eq  0 && {
+                [[ -x  ${root_binary_path[1]}notify-send ]]  && {
+                     notify-send  --urgency=normal   "${src}  updated to v${version}"
```
Comment (aristocratos) [inline bashtop:467]: Whats the availability of `notify-send` over different distributions and future cross platform compatibility?

**Comment by aristocratos** [comment]
Will add the changes in .gitgnore and the ioshields in the readme for now.

File: bashtop (hunk 2 above)
Comment (Jukoo) [reply bashtop:140]: ah yes ! I had done some research to find the same type of fonts I came across the 
**Calvin S** type fonts but I couldn't find the double line ... If you know the type of font, send me the link  i 'll fix that

Comment (aristocratos) [reply bashtop:140]: It's Calvin S but uppercase :)
http://patorjk.com/software/taag/#p=display&f=Calvin%20S&t=UPDATE

File: bashtop (hunk 5 above)
Comment (Jukoo) [reply bashtop:467]: Most GNU/Linux distributions have this command by default 
but the notification appears only if it is available.  
and don't break the script

File: bashtop (hunk 3 above)
Comment (Jukoo) [reply bashtop:146]: You're absolutely right. I forgot that detail.

File: bashtop (hunk 4 above)
Comment (Jukoo) [reply bashtop:449]: i' ll fix that

File: bashtop (hunk 2 above)
Comment (Jukoo) [reply bashtop:140]: thanks  :100:

**Pull request closed without merging**